app/src/androidTest/java/EspressoBaseTest.kt

```diff
@@ -0,0 +1,19 @@
+import androidx.annotation.IdRes
+import androidx.recyclerview.widget.RecyclerView
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.contrib.RecyclerViewActions
+import androidx.test.espresso.matcher.ViewMatchers.withId
+
+open class EspressoBaseTest {
```

Comment:
Myślę, że to mogę zmienić na object.

Reply:
You can't do that, because it won't compile. You can't call singleton constructor, because it is private and single :). The question is: do you need to have `clickRecyclerViewItem` in `EspressoBaseTest`? You have to choose your approach. I think, in future this class will grow very much. Good principle is to have many small classes. You can try to change `clickRecyclerViewItem` to extension function and extract it to e.g. `RecyclerViewTestExtension` file.